LABTASKS.md

```diff
@@ -46,12 +46,12 @@ You should add a new section to the HTML (without changing any of the existing H
 that provides a list of some of the tools we'll be using in this class:
 
 - [`git`](https://git-scm.com/)
-- [GitHub](https://github.com/)
+- [GitHub.com](https://github.com/)
```

Comment:
Good question. It wasn't actually consistent before. I think it might have been written out as GitHub.com in a few places to distinguish that from github.umn.edu, which we used some for a while. We haven't used that in ages, though, so I could just change everything to GitHub.